pkg/api/util/util/db_test.go

```diff
@@ -0,0 +1,330 @@
+// Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Why is this file under /util/util . There is already /util/db_test.go. Hope this is copy accidentally created. please remove